Examples/DeveloperCocoapodsExample/CocoapodsExample/DeeplinkNavigator.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  DeeplinkNavigator.swift
+//  DeepLinkNavigator.swift
```

Comment:
@mmaatttt I suppose this filename should change to match

Reply:
Yes, I was sure I did that but I guess not 🙃 Thanks for catching it

Reply:
only because I hit the same issue where git didn't pickup my rename, sigh